test/fw/ptl/utils/pbs_logutils.py

```diff
@@ -562,6 +579,153 @@ def get_log_files(self, hostname, path, start, end, sudo=False):
 
         return paths
 
+    def _get_syslog_lines(self, hostname=None, n=100, logval=None):
+        """
+        Get the syslog lines to be matched and send them to _log_match()
+
+        :param hostname: the on which syslog files reside
+        :type hostname: str
+        :param n: the number of lines to match
+        :type n: int
+        :param logval: The entity requested, an instance of a
+                        Scheduler, Server or MoM object, or the
+                        string 'tracejob' for tracejob
+        :type logval: str or object
+        :returns Last ``<n>`` lines of logfile for ``Server``,
+                  ``Scheduler``, ``MoM`` or ``tracejob``
+        """
+
+        # if syslog utility is not rsyslogd, throw error
+
+        which_syslog_utility = str(self.pu.get_proc_info(name="rsyslogd"))
+
+        if "rsyslogd" not in which_syslog_utility:
+            raise PbsConfigError(rc=1, rv=None,
+                                 msg="Currently only rsyslog is supported")
+
+        severity = self.du.parse_pbs_config().get("PBS_SYSLOGSEVR")
+
+        if severity is None:
+            raise PbsConfigError(rc=1, rv=None,
+                                 msg="Syslog is requested and "
+                                     "PBS_SYSLOGSVR not set in pbs.conf")
+
+        facility = self.du.parse_pbs_config().get("PBS_SYSLOG")
+
+        list_of_syslog_files = self._get_rsyslog_files(hostname=hostname,
+                                                       severity=severity,
+                                                       facility=facility)
+
+        lines = []
+        if n == "ALL":
+            n = 5000
+        if hostname is None:
+            hostname = socket.gethostname()
+
+        if logval is 'sched_logs':
+            daemon_str = 'pbs_sched'
+        elif logval is 'server_logs':
```

Comment:
here the logval value passed is the the output of instance_to_logpath(). 
_get_syslog_lines() is called in  log_lines() in PBSObject class. Since instance_to_logpath() is a part of PBSObject I call it in log_lines() itself.